### ISSUE TITLE
В розділі "Topics" кнопка для прокручування сторінки вгору, знаходиться в горі сторінки

### ISSUE THREAD

**Issue by Nadiyk** [scholokov/long-travel-2#1387]
Передумови:
Відкрити https://test.long-travel.live/
Кроки для відтворення:
1. Клікніть на розділ "Topics" в горі сторінки з ліва
2. З ліва відображається кнопка для прокрутки сторінки в гору

Фактичний результат: На сторінці розмішена кнопка для прокрутки сторінки в гору
Очікуваний результат: Кнопка неповинна там бути так як сторінка знаходиться в самій горі і її непотрібно прокручувати в гору

Середовище: Windows 10 Pro Операційна
система:
Браузер: Google Chrome Версія
![лишня клавіша](https://user-images.githubusercontent.com/113175298/193087271-9bbc02b4-a626-4e6f-b26d-43699d467611.jpg)

**Comment by DmytroMarkulych** [comment]
@Nadiyk а можливо там цілий розділ не в тому місці що потрібно?)

**Comment by Nadiyk** [comment]
що мені робити з цим багом?


чт, 29 сент. 2022 г. в 20:07, DmytroMarkulych ***@***.***>:

**Comment by DmytroMarkulych** [comment]
@Nadiyk ні, закривати не потрібно, це дефект. Просто якщо на нього подивитися ширше, звернути ще увагу на інші UI елементи, а порівняти із іншими сторінками структуру сторінок. Рекомендую звернути увагу на footer

**Comment by DmytroMarkulych** [comment]
@Nadiyk там footer знаходиться під header. Це хороший дефект, просто його треба розширити

**Comment by Nadiyk** [comment]
![image](https://user-images.githubusercontent.com/113175298/193643174-3d03b908-0b86-4e1a-9bf2-d830458819f8.png)
так, тут в футері є дифекти але я не розумію як їх повязати з цим вже створеним багом

**Comment by DmytroMarkulych** [comment]
@Nadiyk ось це footer https://i.imgur.com/Q1vtvdc.png . і він під header

**Comment by Nadiyk** [comment]
Виправила
